examples/sensors/camera_sensor_example.py

```diff
@@ -0,0 +1,339 @@
+"""
+Example demonstrating camera sensors with different rendering backends.
+
+This example shows:
+1. Creating cameras as sensors using add_sensor() with three backends:
+   - Rasterizer (OpenGL-based, fast for debugging)
+   - Raytracer (LuisaRender, high-quality path tracing) - optional if LuisaRenderPy installed
+   - BatchRenderer (Madrona, efficient multi-camera GPU rendering)
+2. Multiple cameras sharing the same backend renderer
+3. Both .render() and .read() API patterns
+4. Adding lights per backend (each backend has its own light system)
+5. Rendering across multiple environments (batched)
+6. Attaching/detaching cameras to rigid links for ALL backends (dynamic camera mounting)
+   - Cameras follow a falling sphere from step 0-50
+   - At step 50, all cameras detach and remain static
+   - Demonstrates dynamic viewpoint changes vs. static cameras
+7. Comparing rendering quality across different backends
+8. Graceful handling of optional dependencies (LuisaRenderPy)
+"""
+
+import genesis as gs
+from genesis.options.sensors import RasterizerCameraOptions, RaytracerCameraOptions, BatchRendererCameraOptions
+
+########################## init ##########################
+gs.init(seed=0, precision="32", backend=gs.gpu, logging_level="info")
+
+########################## create a scene ##########################
+scene = gs.Scene(
+    rigid_options=gs.options.RigidOptions(
+        enable_collision=True,
+        gravity=(0, 0, -9.8),
+    ),
+    renderer=gs.renderers.RayTracer(
+        env_surface=gs.surfaces.Emission(
+            emissive_texture=gs.textures.ColorTexture(color=(0.2, 0.3, 0.5)),
+        ),
+        env_radius=20.0,
+    ),
+    show_viewer=False,
+)
+
+########################## entities ##########################
+plane = scene.add_entity(
+    morph=gs.morphs.Plane(),
+    surface=gs.surfaces.Rough(color=(0.4, 0.4, 0.4)),
+)
+
+sphere = scene.add_entity(
+    morph=gs.morphs.Sphere(pos=(0.0, 0.0, 2.0), radius=0.5),
+    surface=gs.surfaces.Smooth(color=(1.0, 0.5, 0.5)),
+)
+
+box = scene.add_entity(
+    morph=gs.morphs.Box(pos=(1.0, 1.0, 1.0), size=(0.3, 0.3, 0.3)),
+    surface=gs.surfaces.Rough(color=(0.5, 1.0, 0.5)),
+)
+
+########################## Example 1: Rasterizer Cameras ##########################
+print("\n=== Rasterizer Cameras ===")
+
```

Comment:
I would rather suggest to have an input argument to select the desired type of camera backend. Instantiating them all at once is not something we want to officially support and we should not encourage the user to do so.

Reply:
It's an interesting test to render all camera backend and compare with the visual quality.